pkg/api/handlers/compat/containers.go

```diff
@@ -316,7 +317,7 @@ func LibpodToContainerJSON(l *libpod.Container, sz bool) (*types.ContainerJSON,
 
 	cb := types.ContainerJSONBase{
 		ID:              l.ID(),
-		Created:         l.CreatedTime().String(),
+		Created:         l.CreatedTime().Format(time.RFC3339),
```

Comment:
Think this should be RFC3339Nano instead

Reply:
nice catch @mheon , I'd to double check, that's what Docker displayes to their logs.

Reply:
Thanks @mheon! I expected that regular ISO_8601 format can be used. However changed to:
```
l.CreatedTime().Format(time.RFC3339Nano),
```
Now output is:
```shell
"2020-10-12T08:48:53.855528237+02:00"  
```